utils/test_formats.py

```diff
@@ -117,7 +117,7 @@ def test_intakes_produce_expected_messages(request, manager, intakes_root, test_
 
     expected = testcase["expected"]
 
-    if request.config.getoption("fix_expectations") and parsed != expected:
+    if True: #request.config.getoption("fix_expectations") and parsed != expected:
```

Comment:
I'm not sure that you expect this code to land in the PR.

Reply:
Yes, you're right. Thanks!